.nvmrc

```diff
@@ -0,0 +1 @@
+lts/gallium
```

Comment:
This should probably be `lts/hydrogen` now, since that's the [current Node LTS version](https://nodejs.dev/en/about/releases/).